<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>INFORMATIONOPTIONSDIALOGUI_C</class>
 <widget class="QDialog" name="INFORMATIONOPTIONSDIALOGUI_C">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>329</width>
    <height>611</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Information Options</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/moonphaseqt.qrc">
    <normaloff>:/MoonPhaseQtIcon</normaloff>:/MoonPhaseQtIcon</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="m_pUnitsGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>&amp;Units</string>
     </property>
     <layout class="QVBoxLayout" name="m_pUnitsVLayout"/>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="m_pFormatGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Format</string>
     </property>
     <layout class="QVBoxLayout" name="m_pFormatVLayout">
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="m_pFormatSampleTextFrame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="m_pFormatSampleTextLabel">
           <property name="text">
            <string>Sample text</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="m_pTimeGroupBox">
        <property name="title">
         <string>Time</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QCheckBox" name="m_p24HourFormatCheckBox">
           <property name="text">
            <string>24 &amp;hour format</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="m_pShowSecondsCheckBox">
           <property name="text">
            <string>&amp;Show seconds</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="m_pDateGroupBox">
        <property name="title">
         <string>Date</string>
        </property>
        <layout class="QVBoxLayout" name="m_pDateVLayout">
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QFormLayout" name="m_pStyleFLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="m_pStyleLabel">
             <property name="text">
              <string>&amp;Style</string>
             </property>
             <property name="buddy">
              <cstring>m_pStyleComboBox</cstring>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="m_pStyleComboBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QFormLayout" name="formLayout">
           <item row="1" column="0">
            <widget class="QCheckBox" name="m_pLongMonthFormatCheckBox">
             <property name="text">
              <string>Long &amp;month format</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QCheckBox" name="m_pShowDayOfWeekCheckBox">
             <property name="text">
              <string>Show day of &amp;week</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QCheckBox" name="m_pLongDayOfWeekFormatCheckBox">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>&amp;Long format</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QCheckBox" name="m_p4DigitYearCheckBox">
             <property name="text">
              <string>4 digit &amp;year</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="m_pDisplayGroupBox">
     <property name="title">
      <string>Display</string>
     </property>
     <layout class="QVBoxLayout" name="m_pDisplayVLayout">
      <item>
       <widget class="QFrame" name="m_pDisplaySampleTextFrame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="m_pDisplaySampleTextLabel">
           <property name="text">
            <string>Sample text</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="m_pColorFLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::FieldsStayAtSizeHint</enum>
        </property>
        <item row="2" column="1">
         <widget class="QwwColorButton" name="m_pTextColorButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="showName">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QwwColorButton" name="m_pBackgroundColorButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="showName">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="m_pUseSystemTextColorCheckBox">
          <property name="text">
           <string>Use system &amp;text
color</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QCheckBox" name="m_pUseSystemBackgroundColorCheckBox">
          <property name="text">
           <string>Use system
&amp;background color</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="m_pUseSystemFontCheckBox">
          <property name="text">
           <string>Use system &amp;font</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="m_pChangeFontButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>&amp;Change...</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="m_pMarginLabel">
          <property name="text">
           <string>&amp;Margin</string>
          </property>
          <property name="buddy">
           <cstring>m_pMarginSpinBox</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="m_pMarginSpinBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="m_pButtonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwwColorButton</class>
   <extends>QPushButton</extends>
   <header>qwwcolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/moonphaseqt.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_pChangeFontButton</sender>
   <signal>clicked()</signal>
   <receiver>INFORMATIONOPTIONSDIALOGUI_C</receiver>
   <slot>ChangeFontButtonClickedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>474</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pTextColorButton</sender>
   <signal>colorPicked(QColor)</signal>
   <receiver>INFORMATIONOPTIONSDIALOGUI_C</receiver>
   <slot>TextColorPickedSlot(QColor)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>512</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pBackgroundColorButton</sender>
   <signal>colorPicked(QColor)</signal>
   <receiver>INFORMATIONOPTIONSDIALOGUI_C</receiver>
   <slot>BackgroundColorPickedSlot(QColor)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>556</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pButtonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>INFORMATIONOPTIONSDIALOGUI_C</receiver>
   <slot>ButtonBoxButtonClickedSlot(QAbstractButton*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>601</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_p24HourFormatCheckBox</sender>
   <signal>clicked()</signal>
   <receiver>INFORMATIONOPTIONSDIALOGUI_C</receiver>
   <slot>OptionChangedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pShowSecondsCheckBox</sender>
   <signal>clicked()</signal>
   <receiver>INFORMATIONOPTIONSDIALOGUI_C</receiver>
   <slot>OptionChangedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pStyleComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>INFORMATIONOPTIONSDIALOGUI_C</receiver>
   <slot>OptionChangedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pLongMonthFormatCheckBox</sender>
   <signal>clicked()</signal>
   <receiver>INFORMATIONOPTIONSDIALOGUI_C</receiver>
   <slot>OptionChangedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_p4DigitYearCheckBox</sender>
   <signal>clicked()</signal>
   <receiver>INFORMATIONOPTIONSDIALOGUI_C</receiver>
   <slot>OptionChangedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pShowDayOfWeekCheckBox</sender>
   <signal>clicked()</signal>
   <receiver>INFORMATIONOPTIONSDIALOGUI_C</receiver>
   <slot>OptionChangedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>329</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pLongDayOfWeekFormatCheckBox</sender>
   <signal>clicked()</signal>
   <receiver>INFORMATIONOPTIONSDIALOGUI_C</receiver>
   <slot>OptionChangedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>329</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pMarginSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>INFORMATIONOPTIONSDIALOGUI_C</receiver>
   <slot>OptionChangedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>428</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pUseSystemFontCheckBox</sender>
   <signal>clicked()</signal>
   <receiver>INFORMATIONOPTIONSDIALOGUI_C</receiver>
   <slot>OptionChangedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>458</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pUseSystemTextColorCheckBox</sender>
   <signal>clicked()</signal>
   <receiver>INFORMATIONOPTIONSDIALOGUI_C</receiver>
   <slot>OptionChangedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>499</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pUseSystemBackgroundColorCheckBox</sender>
   <signal>clicked()</signal>
   <receiver>INFORMATIONOPTIONSDIALOGUI_C</receiver>
   <slot>OptionChangedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>543</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ChangeFontButtonClickedSlot()</slot>
  <slot>BackgroundColorPickedSlot(QColor)</slot>
  <slot>TextColorPickedSlot(QColor)</slot>
  <slot>DisplayCheckBoxChangedSlot()</slot>
  <slot>MarginChangedSlot(int)</slot>
  <slot>ButtonBoxButtonClickedSlot(QAbstractButton*)</slot>
  <slot>FormatCheckBoxChangedSlot()</slot>
  <slot>DateStyleCurrentIndexChangedSlot(int)</slot>
  <slot>OptionChangedSlot()</slot>
 </slots>
</ui>
